lib/pbench/test/unit/server/query_apis/test_datasets_update.py

```diff
@@ -238,6 +240,8 @@ def test_query_owner_publish(
         is_admin = build_auth_header["header_param"] == HeaderTypes.VALID_ADMIN
         if not HeaderTypes.is_valid(build_auth_header["header_param"]):
             expected_status = HTTPStatus.UNAUTHORIZED
+        elif not bool(query_json):
+            expected_status = HTTPStatus.BAD_REQUEST
```

Comment:
If you make a change for some other reason, you should remove the `bool()` coercion:  because of the `not` operator (and, because of the `elif`...), we're already in a boolean context, so explicitly coercing the expression to a boolean result doesn't accomplish anything.